src/System.Reflection.TypeExtensions/src/System.Reflection.TypeExtensions.csproj

```diff
@@ -21,7 +22,7 @@
     <Compile Include="System\Reflection\Requires.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' != 'netcore50aot'">
-    <Compile Include="System\Reflection\TypeExtensions.CoreCLR.cs" />
+    <Compile Include="System\Reflection\TypeExtensions.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'netcore50aot'">
```

Comment:
Sounds good.

BTW: In the unlikely event we ever need to service the netcore50 version of this one, there is no way we would do it out of master because of so many things have changed. I expect that it would be done out of release branch.